.github/workflow/unit_test.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
```

Comment:
This is not a build workflow, its a unit test one, update name to reflect that (or remove this line